k8s/hyperconverged/tutorial-configure-openebs-gke.md

```diff
@@ -1,76 +1,132 @@
 
 ### Minimum Requirements
+
 - Machine Type - (Minimum 2 vCPUs)
-- Node Image - (container-vm)
+- Node Image - (Ubuntu)
 - Size - (Minimum 3)
 - Cluster Version - (1.6.4+)
 
 *Note: The example commands below were run on a container cluser named (demo-openebs03) in zone (us-central1-a). The project unique id (strong-eon-153112). When copy/pasting the command, make sure to use the details from your project.*
 
-### Add iSCSI Support
+### iSCSI Configuration
 
-SSH into the nodes of the cluster (Google Cloud Platform --> Compute Engine --> VM instances) to install open-iscsi package. OpenEBS 0.3 uses iscsi to connect to the block volumes.
+Go to **Google Cloud Platform** -> **Compute Engine** -> **VM instances**. The nodes that are displayed by default in this console are the Compute Engine VMs.The display is similar to the following screen.
 
-```
-sudo apt-get update
-sudo apt-get install open-iscsi
-sudo service open-iscsi restart
-```
+![Compute VMs]
 
 #### Verify that iSCSI is configured
 
-Check that initiator name is configured and iscsi service is running
-```
-sudo cat /etc/iscsi/initiatorname.iscsi
-sudo service open-iscsi status
-```
+ OpenEBS uses iSCSI to connect to the block volumes. Select one of the nodes in the cluster and click **SSH**, to open the node in a terminal.
+
+1. Check if the initiator name is configured.
+
+  ```bash
+
+    ~$ sudo cat /etc/iscsi/initiatorname.iscsi
+    ## DO NOT EDIT OR REMOVE THIS FILE!
+    ## If you remove this file, the iSCSI daemon will not start.
+    ## If you change the InitiatorName, existing access control lists
+    ## may reject this initiator.  The InitiatorName must be unique
+    ## for each iSCSI initiator.  Do NOT duplicate iSCSI InitiatorNames.
+    InitiatorName=iqn.1993-08.org.debian:01:6277ea61267f
+
+  ```
+
+2. Check if the iSCSI service is running.
+
+  ```bash
+
+   ~$ sudo service open-iscsi status
+   ● open-iscsi.service - Login to default iSCSI targets
+     Loaded: loaded (/lib/systemd/system/open-iscsi.service; enabled; vendor preset: enabled)
+     Active: active (exited) since Tue 2017-10-24 14:33:57 UTC; 3min 6s ago
+       Docs: man:iscsiadm(8)
+             man:iscsid(8)
+    Main PID: 1644 (code=exited, status=0/SUCCESS)
+       Tasks: 0
+      Memory: 0B
+         CPU: 0
+      CGroup: /system.slice/open-iscsi.service
+   Oct 24 14:33:57 gke-cluster-3-default-pool-8b0f2a27-5nr2 systemd[1]: Starting Login to default iSCSI targets...
+   Oct 24 14:33:57 gke-cluster-3-default-pool-8b0f2a27-5nr2 iscsiadm[1640]: iscsiadm: No records found
+   Oct 24 14:33:57 gke-cluster-3-default-pool-8b0f2a27-5nr2 systemd[1]: Started Login to default iSCSI targets.
+
+  ```
+
+3. Repeat steps 1 and 2 for the remaining nodes.
 
 ## Step 2 : Run OpenEBS Operator (via Google Cloud Shell)
 
 Download the latest OpenEBS Operator Files
-```
+
```

Comment:
Move the steps for download and latest .. to Step 2 -- below after the admin context has been created.